kyo-bench/src/main/scala/kyo/bench/Bench.scala

```diff
@@ -20,7 +21,29 @@ import zio.UIO
     )
 )
 @BenchmarkMode(Array(Mode.Throughput))
-abstract class Bench[T]
+abstract class Bench[T]:
+
+    @Param(Array("false"))
+    var replaceZioExecutor = false
+
+    lazy val zioRuntime =
+        import zio.*
+        if !replaceZioExecutor then
+            zio.Runtime.default.unsafe
+        else
+            given Unsafe = Unsafe.unsafe(identity)
+            val exec =
+                new Executor:
+                    val scheduler                     = kyo.scheduler.Scheduler.get
+                    def metrics(using unsafe: Unsafe) = None
+                    def submit(runnable: Runnable)(implicit unsafe: Unsafe): Boolean =
+                        scheduler.schedule(kyo.scheduler.Task(runnable.run()))
+                        true
+            val kExecutorLayer = Runtime.setExecutor(exec) ++ Runtime.setBlockingExecutor(exec)
+            Runtime.unsafe.fromLayer(kExecutorLayer).unsafe
```

Comment:
We may need to avoid these 'layer' based operators and use the `zio` namespace to manually invoke Runtime creation

Reply:
Yeah, I wonder if the fact that we're using layers for the config is what generating the itable stub overhead